src/Orleans/Runtime/GrainReference.cs

```diff
@@ -687,6 +714,11 @@ protected GrainReference(SerializationInfo info, StreamingContext context)
             if (String.IsNullOrEmpty(genericArg))
                 genericArg = null;
             genericArguments = genericArg;
+
+#if !NETSTANDARD
+            var deserializationContext = context.Context as IDeserializationContext;
+            deserializationContext?.GrainFactory.BindGrainReference(this);
+#endif
```

Comment:
While netstandard1.5 has a StreamingContext type, it's a stub and serves no real purpose other than to exist